### ISSUE TITLE
computation of negative powers of a Quaternion

### ISSUE THREAD

**Issue by RohitJain3241** [sympy/sympy#13688]
Computation of negative powers of a Quaternion returns 1 as result. Is this correct??
I think q^p =  (q.pow(-1)).pow(p). Am I correct? @asmeurer @Upabjojr

**Comment by ArighnaIITG** [comment]
I can work on this.
The pow function in quaternion.py is not defined for p<-1 . res=1 is defined for p<-1.
Is that mathematically correct?
@gxyd  @asmeurer

**Comment by RohitJain3241** [comment]
I am waiting for @asmeurer to confirm this. Then maybe we could work on this.

**Comment by ArighnaIITG** [comment]
@gxyd , @jksuom , @asmeurer 
Please review this issue, and please say if this idea is correct.
If yes, What will I implement for p<-1??

**Comment by jksuom** [comment]
Negative powers can be defined as (positive) powers of the inverse provided that exists. (So, if `p < 0`, then try to replace `q` by `q.inverse()`, and if that succeeds, then continue with the positive exponent `-p`.)

**Comment by Vinayak-Shukla** [comment]
Is the issue still open? I'd like to work on it please.

**Comment by oscarbenjamin** [comment]
The issue is still open although #13745 says it fixes this and has been merged. This didn't get automatically closed because the PR description says "This fixes #..." rather than "Fixes #...". Probably the reviewer forgot to manually close this.

We should check that the issue is actually solved before closing though:
```julia
In [3]: q = Quaternion(1, 2, 3, 4)                                                                                                                            

In [4]: q                                                                                                                                                     
Out[4]: 1 + 2*i + 3*j + 4*k

In [5]: q**-1                                                                                                                                                 
Out[5]: 1/30 + (-1/15)*i + (-1/10)*j + (-2/15)*k

In [6]: q * q**-1                                                                                                                                             
Out[6]: 1 + 0*i + 0*j + 0*k

In [7]: q**2                                                                                                                                                  
Out[7]: (-28) + 4*i + 6*j + 8*k

In [8]: q**-2                                                                                                                                                 
Out[8]: (-7/225) + (-1/225)*i + (-1/150)*j + (-2/225)*k

In [9]: q**2 * q**-2                                                                                                                                          
Out[9]: 1 + 0*i + 0*j + 0*k
```

Looks like it's fixed and this should have been closed by #13745. Closing.

**Comment by jksuom** [comment]
I was fixed by #13745. Closing.
